### PR TITLE
Some work on JobPostings and app.jsx

### PR DESCRIPTION
**Pull request by ravenf12**

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Auth from "./components/auth/Auth";
 import AuthCompany from "./components/auth/AuthCompany";
 import Profile from "./components/profile/Profile";
 import LandingPage from "./components/LandingPage/LandingPage";
-import JobPostings from "./components/auth/jobPostings/JobPostings";
+import JobPostings from "./components/jobPostings/JobPostings";
```
Comment (ravenf12) [inline client/src/App.jsx:6]: Pulling jobPostings out of auth

**Review by WillMashrick — APPROVED** [review]

File: client/src/App.jsx [hunk 2]
```diff
@@ -24,23 +24,14 @@ function App() {
...
-        {/* <Route path="/" element={<Auth updateToken={updateToken} />} /> */}
         <Route path="/" element={<LandingPage />} />
         <Route path="/start" element={<Auth updateToken={updateToken} />} />
         <Route
           path="/companystart"
           element={<AuthCompany updateToken={updateToken} />}
         />
         <Route path="/profile" element={<Profile />} />
-        <Route path="/jobs" element={<JobPostings token={sessionToken} />} /> 
+        <Route path="/jobPostings" element={<JobPostings token={sessionToken} />} />
```
Comment (ravenf12) [inline client/src/App.jsx:31]: Change to path for jobPostings

**Review by WillMashrick — APPROVED** [review]

File: client/src/App.jsx [hunk 3]
```diff
@@ -24,23 +24,14 @@ function App() {
...
-        <Route path="/job" element={<JobPostings />}/>
-        {/* <Route path="/" element={<Auth updateToken={updateToken} />} /> */}
         <Route path="/" element={<LandingPage />} />
         <Route path="/start" element={<Auth updateToken={updateToken} />} />
         <Route
           path="/companystart"
           element={<AuthCompany updateToken={updateToken} />}
         />
         <Route path="/profile" element={<Profile />} />
-        <Route path="/jobs" element={<JobPostings token={sessionToken} />} />
```
Comment (ravenf12) [inline client/src/App.jsx:43]: Delete me

File: client/src/App.jsx
Comment (ravenf12) [inline client/src/App.jsx:1]: done

File: client/src/components/jobPostings/JobPostings.jsx
Comment (ravenf12) [inline client/src/components/jobPostings/JobPostings.jsx:1]: done

**Review by WillMashrick — APPROVED** [review]

**Review by ravenf12 — COMMENTED** [review]
These are all of the changes I needed to make, Hoping the file will delete itself once posting a comment.

**Review by WillMashrick — APPROVED** [review]

**Pull request closed without merging**